### ISSUE TITLE
Crash on Start

### ISSUE THREAD

**Issue by averaldofh** [tyrope/DVSaveSwitcher#3]
I've tried the .exe option. First run ok, creates config file and closes.
After that it crash right after start, with no messages. 
So, i tried the .py option. No success. same behaviour
then I copied the code to IDLE and ran it, then i could record the errors, and this is what i get:

Traceback (most recent call last):
  File "C:\Users\avera\Downloads\DVSaveSwitcher-127-1-0-4-1598106261\dvss.py", line 225, in <module>
    while process_exists('derailValley.exe'):
  File "C:\Users\avera\Downloads\DVSaveSwitcher-127-1-0-4-1598106261\dvss.py", line 197, in process_exists
    output = subprocess.check_output(call).decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 7: invalid start byte

I commented lines 225 to 230 and it seems to run ok. But now there's no check if the game is running. not a problem for me, but maybe could help other users if you figure out what happened. I would try, but i dont know nothing of python. :/
I'm running on windows 10, last build, python 3.9.

**Comment by tyrope** [comment]
Thank you for bringing this bug to my attention! Line 197 seems to be the culprit here...
To quote https://github.com/joho/7XX-rfc
   o  732 - Fucking Unic&#128169;de

**Comment by tyrope** [comment]
Uploading 1.0.5 right now, please let me know if it's not fixed!

Additionally, which language is your Windows install? I ask because it appears to not reproduce on my English machine and I'm wondering if it's a non-latin alphabet that might cause this.

**Comment by averaldofh** [comment]
Sorry for the late response, i downloaded 1.0.5 (from nexus), same issue, first run ok, then crash everytime. I'm using windows in PT-BR. but this time i could run the PY file from this repo with no issues. maybe something when building the .exe, or something wrong with my PC as no one else seems to have this problem

**Comment by averaldofh** [comment]
I used pyinstaller to build an EXE out of the PY from the last release. and now it works fine. Something is wrong with the EXE file indeed..

**Comment by averaldofh** [comment]
i forked the code and add an exe file if you want to try something. don't know if it helps

**Comment by tyrope** [comment]
Hello @averaldofh,

Sorry to hear you're still having issues. My build process is a single command, which is quite minimal:
`pyinstaller --onefile DVSaveSwitcher.py`

Did you build using a different builder, or different parameters?

**Comment by tyrope** [comment]
I added some debug prints, could you try running the above commit and copy the new debug block here?

**Comment by averaldofh** [comment]
Sorry the late responde. I was working on a project. Actually I was learning python.. I used pyinstaller, just the same command you did. I've ran the above commit with no issues.. but I only found the .py file on the commit, the problem I was getting was with the .exe you had on the official branch.

**Comment by averaldofh** [comment]
and while i'm learning a bit of python, i'm now experimenting with tkinter and I'll probably make a GUI for testing. haha

**Comment by tyrope** [comment]
No worries about the late response! The new commit shouldn't actually fix anything, it just shows more information towards the end user. I'm hoping to learn from it too! :)

**Comment by averaldofh** [comment]
allright. I ran the commit again. it printed the following:

_

> ---- DEBUG BLOCK ----
> INFORMAES: nenhuma tarefa em execuo correspondente aos critrios
> especificados.
> 
> especificados.
> ---- DEBUG ENDS  ----

_

which in English means: INFORMATION: No running task corresponding to the specified criteria. [\n\n] specified.

But its running as normal. I guess there was something related to building the EXE. Some variable of the operating system must have caused an issue..but again, I'm not a programmer I'm just trying to help. :)

**Comment by averaldofh** [comment]
ok.. i'm getting something.. at least i think so.. 
i've downloaded the 1.0.4 .py and it gave me the unicode error. That was addressed in 1.0.5, BUT, do you had compiled the .exe from 1.0.5? because the downloaded exe for 1.0.5 gave me the same unicode error that before, while the .py runs smoothly. Here's the terminal output for the 104 code, the 105, and then the 105 EXE.

![terminal](https://user-images.githubusercontent.com/39417988/111088967-214b2880-8500-11eb-84ba-2e0818a01e7b.png)

**Comment by tyrope** [comment]
```D:\Programming\DVSaveSwitcher>pyinstaller --onefile DVSaveSwitcher.py
Traceback (most recent call last):
  File "D:\Python\Scripts\pyinstaller-script.py", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'```
You have succesfully located the bug! New 1.0.5 executable en-route. :)

**Comment by tyrope** [comment]
Resolved with v1.0.6
